README.md

```diff
@@ -3,17 +3,16 @@
 ## Software setup
 
 ```
-cmsrel CMSSW_11_0_0_patch1 
-cd CMSSW_11_0_0_patch1/src
+cmsrel CMSSW_11_1_2 
+cd CMSSW_11_1_2/src
 cmsenv
 
 setenv CMSSW_GIT_REFERENCE /cvmfs/cms.cern.ch/cmssw.git.daily
 git cms-init
 
 git cms-addpkg RecoBTag
-git cms-merge-topic emilbols:BTV_11_0_X
```

Comment:
you need here instead:

git cms-merge-topic emilbols:BTV_CMSSW_11_1_X

It replaces the Phase1 DeepJet training with the phase2 one, and ads the phase2 deepcsv training

Reply:
@emilbols  The branch already had the topic **emilbols:BTV_CMSSW_11_1_X** merged. 
I have updated the instructions in the README (see commit below) and in the twiki (https://twiki.cern.ch/twiki/bin/view/CMS/BTagAnalyzer#Recipe_for_Phase2_11_1_X)